layouts/shortcodes/ci-autoinstrumentation.md

```diff
@@ -0,0 +1,14 @@
+We support auto-instrumentation for the following CI providers:
+| CI Provider | Auto-Instrumentation method |
+|---|---|
+| GitHub Actions | [Datadog Test Visibility Github Action][1] |
+| Jenkins | [UI-based configuration][2] |
+| GitLab | [Datadog Test Visibility GitLab Script][3] |
+| CircleCI | [Datadog Test Visibility CircleCI Orb][4] |
+
+If you are using autoinstrumentation for one of these providers, the rest of the setup steps below can be skipped.
+
+[1]: https://github.com/marketplace/actions/configure-datadog-test-visibility
+[2]: /continuous_integration/pipelines/jenkins/#enable-with-the-jenkins-configuration-ui-1
+[3]: https://github.com/DataDog/test-visibility-gitlab-script
+[4]: https://circleci.com/orbs/registry/orb/datadog/test-visibility-circleci-orb
```

Comment:
404s currently, but I'm guessing this is cos it hasn't been released yet?

Reply:
Yes, exactly :)